### PR TITLE
21

### PR DESCRIPTION
**Pull request by cichan02**

### CODE REVIEW

**Comment by cichan02** [comment]
First of all merge https://github.com/cqqqt/Task-Viewer-Backend/pull/49

File: src/test/java/com/taskviewer/api/postgres/PgTasksIntegrationTest.java [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+
+@SpringBootTest(classes = TaskViewerApplication.class)
+@ExtendWith(SpringExtension.class)
+@TestPropertySource("classpath:application-test.yaml")
+public class PgTasksIntegrationTest extends TestcontainersTest {
+
+	@Autowired
+	private PgTasks pgTasks;
```
Comment (h1alexbel) [inline src/test/java/com/taskviewer/api/postgres/PgTasksIntegrationTest.java:21]: use indentation spaces: 2

File: src/test/resources/application-test.yaml [hunk 2]
```diff
@@ -0,0 +1,9 @@
+spring:
+  datasource:
+    url: ${TEST_URL}
+    username: ${TEST_USERNAME}
+    password: ${TEST_PASSWORD}
+    driver-class-name: org.testcontainers.jdbc.ContainerDatabaseDriver
+  jpa:
+    hibernate:
+      ddl-auto: create
```
Comment (h1alexbel) [inline src/test/resources/application-test.yaml:9]: @cichan02 we are not using any ORM here

we manage our schema using Liquibase instead

**Review by h1alexbel — CHANGES_REQUESTED** [review]

**Comment by cichan02** [comment]
Please view only https://github.com/cqqqt/Task-Viewer-Backend/pull/48/commits/1403df8766ed72e24df53044ff240a7c5b1898ef, because other one is from #18

**Review by h1alexbel — COMMENTED** [review]
@cichan02 use 2 space indentation instead of 4

**Comment by h1alexbel** [comment]
@cichan02 please fix the merge conflict

**Comment by cichan02** [comment]
@h1alexbel merge https://github.com/cqqqt/Task-Viewer-Backend/pull/56

**Review by h1alexbel — APPROVED** [review]

**Comment by h1alexbel** [comment]
@cichan02 fix merge conflict

**Comment by cichan02** [comment]
image

**Comment by h1alexbel** [comment]
@cichan02 just keep one, they're equal

**Review by h1alexbel — APPROVED** [review]

**Comment by h1alexbel** [comment]
@cichan02 fix merge conflicts

**Comment by cichan02** [comment]
@h1alexbel fixed all merge conflicts

**Comment by cichan02** [comment]
@h1alexbel solved all merge conflict

**Review by h1alexbel — CHANGES_REQUESTED** [review]
@cichan02 use 2 space indentation

**Comment by cichan02** [comment]
Screenshot 2023-04-11 at 14 38 38
Screenshot 2023-04-11 at 14 38 38
@h1alexbel see...

**Review by h1alexbel — APPROVED** [review]

**Comment by cichan02** [comment]
Hooray 🥳 for me!

**Comment by h1alexbel** [comment]
@cichan02 thanks!

**Pull request merged**